proposed/2023/Add-dotnet-search.md

```diff
@@ -24,15 +24,18 @@ Imagine you have your dotnet CLI open. You would like to look up a NuGet package
 
 The `package search [search terms] [options]` command will have the following options
 
-| Option | Function |
-|---------|:----------|
-| `--source` | A source to search |
-| `--exact-match` | Return exact matches only as a search result |
-| `--prerelease` | Allow prerelease packages to be shown. |
-| `--interactive` | Allows the command to stop and wait for user input or action (for example to complete authentication).|
-| `--take` | The number of results to return. The default value is 20.|
-| `--skip` | The number of results to skip, for pagination. The default value is 0. |
-| `--help` | Show command help and usage information |
+| Option           | Function |
+|------------------|--------------|
+| `--configFile`   | The path to the NuGet config file to use. |
```

Comment:
here or in a configFile section below can it explain how it finds the config file if you do not pass this flag ? presumably it should also say it only uses exactly one config file.

Reply:
oh, doh, I thought this md was going to be the doc...

Reply:
Here is the PR I made for the public documentation: https://github.com/dotnet/docs/pull/37747.  I will be continuously updating it and will only be merged once the command is released. I will make sure to explain it in the doc.